elastic/logs/challenges/auto-follow-pattern.json

```diff
@@ -0,0 +1,47 @@
+{% import "rally.helpers" as rally %}
+{% set p_follow_index_pattern = "copy-{{leader_index}}" %}
+{
+  "name": "auto-follow-pattern",
```

Comment:
Maybe we can use a better name for the challenge...something that reminds what this does? Something like "ccr-shard-recovery". I guess this track is not only useful for synthetic source... but probably it would provide useful info in Serverless too where we probably retrieve data from S3?

Reply:
> Maybe we can use a better name for the challenge...something that reminds what this does?

+1 - I will add ccr replication to the name and add a description.

> . I guess this track is not only useful for synthetic source... but probably it would provide useful info in Serverless too where we probably retrieve data from S3?

IIRC CCR currently only works in stateful. To keep the follower indices up to date it relies on operation based replication, which doesn't exist in serverless.